iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -10,51 +10,44 @@
 using Microsoft.Azure.Devices.Generated;
 using Microsoft.Rest;
 using Newtonsoft.Json;
+using Microsoft.Azure.Devices.DigitalTwin.Authentication;
+using Azure;
+using Azure.Core;
+using PnpDigitalTwin = Microsoft.Azure.Devices.Generated.DigitalTwin;
 
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// The Digital Twins Service Client contains methods to retrieve and update digital twin information, and invoke commands on a digital twin device.
+    /// For more information, see <see href="https://github.com/Azure/azure-iot-sdk-csharp#iot-hub-service-sdk"/>
     /// </summary>
     public class DigitalTwinClient : IDisposable
```

Comment:
Did we not need to regenerate to reference the latest API version?

Reply:
This is not in the generated folder. Am I missing something?

Reply:
I don't know if AAD auth is tied to an API version, and I wondered if it was necessary to increment it to latest.

I did it for the legacy clients for parentScopes, but didn't realize the DT client has it hardcoded from autogeneration. This was the closest file for me to ask this question.

Reply:
Got your question. Good question but all the tests passed with the current version. I am guessing it was not implemented for a specific API version. The API surface did not really change, only the token we send in the header is a different one so they must have been able to do it without changing the API version.